plugins/quickicon/overridecheck/overridecheck.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * @package     Joomla.Plugin
+ * @subpackage  Quickicon.Overridecheck
+ *
+ * @copyright   Copyright (C) 2005 - 2018 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+defined('_JEXEC') or die;
+
+use Joomla\CMS\Uri\Uri;
+use Joomla\CMS\Factory;
+use Joomla\CMS\Language\Text;
+use Joomla\CMS\Session\Session;
+use Joomla\CMS\HTML\HTMLHelper;
+use Joomla\CMS\Plugin\CMSPlugin;
+
+/**
+ * Joomla! update notification plugin
```

Comment:
Change 
Joomla! update notification plugin
in
Joomla! template override notification plugin?